boards/arm/particle_xenon/particle_xenon_defconfig

```diff
@@ -0,0 +1,25 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF=y
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_PARTICLE_XENON=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable UART driver, needed for CDC ACM
+CONFIG_SERIAL=y
+CONFIG_UART_NRFX=y
+CONFIG_UART_0_NRF_UARTE=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_RTT_CONSOLE=y
+
+# additional board options
+CONFIG_GPIO_AS_PINRESET=y
+
+# bluetooth
```

Comment:
The remark is probably wider than this PR, but why Bluetooth should be enabled by default on nRF52840 boards? The radio on this SoC also support other protocols, and enabling require to explicitly disable bluetooth otherwise the app just doesn't work.

IMHO enabling Bluetooth should be done at the app/sample level.